src/refract/InfoElementsUtils.h

```diff
@@ -0,0 +1,51 @@
+//
+//  refract/InfoElementsUtils.h
+//  librefract
+//
+//  Created by Jiri Kratochvil on 10/04/2018
+//  Copyright (c) 2018 Apiary Inc. All rights reserved.
+//
+
+#ifndef REFRACT_INFO_ELEMENTS_UTILS_H
+#define REFRACT_INFO_ELEMENTS_UTILS_H
+
+#include "InfoElements.h"
+#include "TypeQueryVisitor.h"
+#include "Element.h"
+
+namespace refract
+{
+
+    template <typename ValueElementType, typename DSDType>
+    void AppendInfoElement(InfoElements& ie, const std::string& key, DSDType&& value)
```

Comment:
I find it very hard to see what this function is doing by looking at it's name, signature and template arguments. Please either add a full doxygen-style comment, or modify the name/signature significantly.